DuckDuckGo/AppDelegate.swift

```diff
@@ -512,14 +512,21 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     }
 
     private func stopTunnelAndShowThankYouMessagingIfNeeded() {
+        if AccountManager().isUserAuthenticated {
+            tunnelDefaults.vpnEarlyAccessOverAlertAlreadyShown = true
+            return
+        }
```

Comment:
This is the change that fixes all of this - the issue was caused by this modal trying to appear during the setup flow, but the alert couldn't be presented so we were never able to set this flag. Now we make sure to set this flag if this is attempted any time after the user already has an account.